utils/src/server/rpc_module.rs

```diff
@@ -390,26 +235,28 @@ impl SubscriptionSink {
 mod tests {
 	use super::*;
 	#[test]
-	fn rpc_context_modules_can_merge_with_rpc_module() {
-		// Prove that we can merge an RpcContextModule with a RpcModule.
+	// TODO: does it make sense to merge two RpcModules with the same context? Previously this worked because an
```

Comment:
sounds reasonable to me to allow merging to modules with the same context but not with different context.

the proper way is to add the module(s) to a server I suppose.